features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/conf/locales/jaggery/locale_default.json

```diff
@@ -379,6 +382,7 @@
     "Password": "Password",
     "Please select at least one subscription tier.": "Please select at least one subscription tier.",
     "Please select at least one transport.": "Please select at least one transport.",
+    "Please select at least one API Security.": "Please select at least one API Security.",
```

Comment:
shouldn't it be at least one API Security method? I am not sure the context this is used in, can you share a UI screenshot to understand better?

Reply:
Yes. That should be API security method. I have attached the modified UI.
![screenshot from 2018-10-12 17-14-07](https://user-images.githubusercontent.com/4003116/46867351-4ffdda00-ce42-11e8-8a87-51c2118f9285.png)